editor/gui/editor_run_bar.h

```diff
@@ -109,6 +111,9 @@ class EditorRunBar : public MarginContainer {
 
 	Button *get_pause_button() { return pause_button; }
 
+	void add_child_control(Control *p_control);
+	void remove_child_control(Control *p_control);
```

Comment:
I don't like the names of these methods, they feel too close to `Node::add_child`.

Would it be better to expose `main_hbox` via some `get_buttons_container` method? Then I could just do:

```C++
EditorRunBar::get_singleton()->get_buttons_container()->add_child(p_control);
```

And reuse the existing `Node` API to add/remove children.

Reply:
Oh, I thought I commented on this long ago...

Yeah, I think going for the single method that returns you the direct parent is a better option. And it seems to be the one that we use for a lot of other such containers (see the mappings in `EditorPlugin::add_control_to_container`). That also resolves your other comment.